### PR TITLE
Added the whoAmI method to the Kuzzle object

### PR DESCRIPTION
**Pull request by xbill82**

### CODE REVIEW

File: source/includes/_kuzzleObject.md [hunk 1]
```diff
@@ -1012,6 +1012,80 @@ Returns the `Kuzzle` object to allow chaining.
...
+kuzzle.checkTokenPromise(token)
+  .then(res => {
+    // ...
+  });
+```
+
+```java
+// Not implemented yet
+```
+
```
Comment (scottinet) [inline source/includes/_kuzzleObject.md]: Could you add an output example?

File: source/includes/_kuzzleObject.md [hunk 2]
```diff
@@ -1012,6 +1012,80 @@ Returns the `Kuzzle` object to allow chaining.
...
+kuzzle.whoAmIPromise()
+  .then(res => {
+    // ...
+  });
+```
+
+```java
+// Not implemented yet
+```
+
```
Comment (scottinet) [inline source/includes/_kuzzleObject.md]: Could you add an output example?

File: source/includes/_kuzzleObject.md (hunk 1 above)
Comment (xbill82) [inline source/includes/_kuzzleObject.md]: :+1:

File: source/includes/_kuzzleObject.md (hunk 2 above)
Comment (xbill82) [inline source/includes/_kuzzleObject.md]: :

**Comment by scottinet** [comment]
Nice! +1

**Pull request merged**